### ISSUE TITLE
Fix setting of bl_mynn_tkebudget in GFS_typedefs.F90 (affects diagnostics only)

### ISSUE THREAD

**Issue by climbfuji** [NOAA-EMC/ufsatm#251]
@egrell found a bug related to setting the `bl_mynn_tkebudget` control variable in `GFS_typedefs.F90`. In particular, the `bl_mynn_tkebudget` variable is not read in via namelist nor set. This should only affect diagnostic output of the MYNN PBL scheme.

To fix, simply add a couple of lines:

Add the `bl_mynn_tkebudget` around line 3446 (https://github.com/NOAA-EMC/fv3atm/blob/ddb56fe83ee75b62bcc9dd03f74228f569a76ff0/ccpp/data/GFS_typedefs.F90#L3446).
Add Model%bl_mynn_tkebudget = bl_mynn_tkebudget around line 4038 (https://github.com/NOAA-EMC/fv3atm/blob/ddb56fe83ee75b62bcc9dd03f74228f569a76ff0/ccpp/data/GFS_typedefs.F90#L4038).

This bug affects both the SCM and FV3 versions of the `GFS_typedefs.F90` file. Ideally, it should be fixed in both repositories simultaneously. The corresponding issue for the CCPP SCM is https://github.com/NCAR/ccpp-scm/issues/238, and the text in this fv3atm issue here was mostly stolen from @grantfirl, who created the SCM issue.

While fixing this issue, other MYNN PBL/SFC related variables should be checked, too.

**Comment by climbfuji** [comment]
@joeolson42 FYI

**Comment by joeolson42** [comment]
This TKE budget option has been kept "hidden" because the diagnostic output is somewhat broken in the current code. I have some fixes coming. I'll make sure the namelist option are straightened out as I finish that work.

**Comment by climbfuji** [comment]
@joeolson42 Found this very old issue. Has this been fixed in the new MYNN code?

**Comment by joeolson42** [comment]
> [@joeolson42](https://github.com/joeolson42) Found this very old issue. Has this been fixed in the new MYNN code?

Yes, this has been fixed in the MYNN-EDMF submodule, but the submodule hasn't yet been hooked up to CCPP.

**Comment by climbfuji** [comment]
> > [@joeolson42](https://github.com/joeolson42) Found this very old issue. Has this been fixed in the new MYNN code?
> 
> Yes, this has been fixed in the MYNN-EDMF submodule, but the submodule hasn't yet been hooked up to CCPP.

Ok, thanks! Good to know we can close this when the MYNN-EDMF submodule finds its way into the UFS ecosystem.
